lib/ansible/modules/network/vyos/vyos_lldp_global.py

```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#############################################
+#                WARNING                    #
+#############################################
+#
+# This file is auto generated by the resource
+#   module builder playbook.
+#
+# Do not edit this file manually.
+#
+# Changes to this file will be over written
+#   by the resource module builder.
+#
+# Changes should be made in the model used to
+#   generate this file or in the resource module
+#   builder template.
+#
+#############################################
+
+"""
+The module file for vyos_lldp_global
+"""
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'network'
+}
+
+DOCUMENTATION = """
+---
+module: vyos_lldp_global
+version_added: 2.9
+short_description: Manage link layer discovery protocol (LLDP) attributes on VyOS devices..
+description: This module manages link layer discovery protocol (LLDP) attributes on VyOS devices.
+author:
+   - Rohit Thakur (@rohitthakur2590)
+options:
+  config:
+    description: The provided link layer discovery protocol (LLDP) configuration.
+    type: dict
+    suboptions:
+      enable:
+        description:
+          - This argument is a boolean value to enable or disable LLDP.
+        type: bool
+      address:
+        description:
+          - This argument defines management-address.
+        type: str
+      snmp:
+        description:
+          - This argument enable the SNMP queries to LLDP database.
+        type: str
+      legacy_protocols:
+        description:
+          - List of the supported legacy protocols.
+        type: list
+        choices:
+          - cdp
+          - edp
+          - fdp
+          - sonmp
+  state:
```

Comment:
Add a note mentioned tested version and connection type used with this module and point to vyos platform guide.
Refer: https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/network/junos/junos_interfaces.py#L106-L107

Applicable for other vyos resource modules as well.